de.metas.adempiere.adempiere/base/src/main/java/de/metas/adempiere/service/ILocationBL.java

```diff
@@ -73,13 +74,13 @@
 	 * Build address string based on given locationId and bpartner and user blocks
 	 * 
 	 * task FRESH-119: transaction no longer needed in this method. Provide the location as object, not by its ID. This way we avoid unnecessary extra loading of the object based on id.
-	 * 
+	 * @param bpartner
```

Comment:
pls document if it's an optional or mandatory parameter. I guess it's optional... 
Also, why adding it as first parameter instead of adding it somewhere near the "bPartnerBlock".